app/interfaces/controllers/error.go

```diff
@@ -9,3 +9,9 @@ func NewError(err error) *Error {
 		Message: err.Error(),
 	}
 }
+
+func ValidationError(info string, err error) *Error {
+	return &Error{
+		Message: "ValidationError about " + info + ":" + err.Error(),
```

Comment:
change about to "for".
And since this is a message already, add a space "Validation Error".

Reply:
Thanks! I changed message.